crates/phala-trie-storage/src/lib.rs

```diff
@@ -87,11 +87,10 @@ where
 
     /// Apply storage changes calculated from `calc_root_if_changes`.
     pub fn apply_changes(&mut self, root: H::Out, transaction: MemoryDB<H>) {
-        self.0.backend_storage_mut().consolidate(transaction);
```

Comment:
@kvinwang

Reply:
forgot to port Kevin's change, will do

Reply:
his port from https://github.com/Phala-Network/phala-blockchain/commit/8fb04ac61aecbeb98a7074c6e06cd8e34528d96d